Core/CoreTests/Views/CoreWebViewTests.swift

```diff
@@ -192,4 +204,8 @@ class CoreWebViewTests: CoreTestCase {
         XCTAssertEqual(CoreWebView.htmlString("html"), "html")
         XCTAssertEqual(CoreWebView.htmlString("&'\"<>"), "&amp;&#39;&quot;&lt;&gt;")
     }
+
+    func testIsLinkNavigationEnabled() {
+
+    }
```

Comment:
Is this supposed to be blank?

Reply:
Nope, forgot I started down that path. Thanks.